docs/src/plans/index.md

```diff
@@ -47,4 +47,10 @@ The column “generic” refers to a short hand that might be used for readabil
 Since the iterate is often stored in the states fields `s.p` one _could_ access the iterate
 often also with `:p` and similarly the gradient with `:X`.
 This is discouraged for both readability as well as to star more generic, and it is recommended
-to use `:Iterate` and `:Gradient` instead in generic settings.
\ No newline at end of file
+to use `:Iterate` and `:Gradient` instead in generic settings.
+
+YOu can further activate a “Tutorial” mode by `set_manopt_parameter!(:Mode, "Tutorial"). Internally, the following convenience function is available.
```

Comment:
```suggestion
You can further activate a “Tutorial” mode by `set_manopt_parameter!(:Mode, "Tutorial")`. Internally, the following convenience function is available.
```

Reply:
Good that I am starting to use vale, that also reports on these. Somehow my VS Code Spellchecker does not want to work as I think it should.